run_unittests.py

```diff
@@ -3198,7 +3198,6 @@ def test_old_gnome_module_codepaths(self):
         self.build()
         mesonbuild.modules.gnome.native_glib_version = None
 
-    @unittest.skipIf(shutil.which('pkg-config') is None, 'Pkg-config not found.')
```

Comment:
Do we really want to hard depend on pkg-config for the test suite on *nix now? I feel that this does not belong into this PR.

Reply:
I wanted to ensure that the test would not be skipped on the CI just because pkg-config wasn't available. I'll change it to skip it when we're not on CI.

Reply:
Please do. I think completely removing these skips should be the topic of another PR.

Reply:
Done.